src/utils/testUtils.ts

```diff
@@ -0,0 +1,8 @@
+import {build, fake, sequence} from 'test-data-bot'
+
+export const postBuilder = build('Post').fields({
```

Comment:
As this is an util function, I think it would be better named as `buildPost`

Reply:
Builder is a convention name